opentelemetry-sdk/benches/metric.rs

```diff
@@ -53,9 +53,70 @@ impl MetricReader for SharedReader {
     }
 }
 
-fn bench_counter(view: Option<Box<dyn View>>) -> (Context, SharedReader, Counter<u64>) {
+/// Configure delta temporality for all [InstrumentKind]
+///
+/// [Temporality::Delta] will be used for all instrument kinds if this
+/// [TemporalitySelector] is used.
+#[derive(Clone, Default, Debug)]
+pub struct DeltaTemporalitySelector {
+    pub(crate) _private: (),
+}
+
+impl DeltaTemporalitySelector {
+    /// Create a new default temporality selector.
+    pub fn new() -> Self {
+        Self::default()
+    }
+}
+
+impl TemporalitySelector for DeltaTemporalitySelector {
+    fn temporality(&self, _kind: InstrumentKind) -> Temporality {
+        Temporality::Delta
+    }
+}
+
+// * Summary *
+
+// rustc 1.68.0 (2c8cc3432 2023-03-06)
+// cargo 1.68.0 (115f34552 2023-02-26), OS=Windows 11 Enterprise
+// Intel(R) Core(TM) i7-8850H CPU @ 2.60GHz   2.59 GHz
+// 12 logical and 6 physical cores
+
+// Counter/AddNoAttrs      time:   [65.406 ns 65.535 ns 65.675 ns]
```

Comment:
qn: what does the 3 set of numbers indicatr? Avg/Min/Max?

Reply:
https://bheisler.github.io/criterion.rs/book/user_guide/command_line_output.html#time